tutorials/epochs/40_autogenerate_metadata.py

```diff
@@ -300,22 +310,40 @@
 )
 
 # %%
-# Lastly, let's visualize the ERPs associated with the visual stimulation, once
-# for all trials with correct responses, and once for all trials with correct
-# responses and a response time greater than 0.5 seconds
-# (i.e., slow responses).
+# You probably also noticed that 9 epchs were dropped because they exceeded our
+# rejection limits. This is another reason why it is important to assign the metadata on
+# epochs creation: the metadata will be updated automatically to reflect the actual
+# epochs that were kept.
+#
+# Lastly, let's visualize the ERPs associated with the visual stimulation. We will only
+# consider trials with correct responses and produce three plots:
+# one for all correct responses, one for correct slow responses (response time slower
+# than 0.5 s), and one for correct fast responses (response time up to 0.5 s).
+
 vis_erp = epochs["response_correct"].average()
-vis_erp_slow = epochs["(not response_correct) & (response > 0.3)"].average()
+vis_erp_slow = epochs["(response_correct) & (response > 0.5)"].average()
+vis_erp_fast = epochs["(response_correct) & (response <= 0.5)"].average()
 
-fig, ax = plt.subplots(2, figsize=(6, 6), layout="constrained")
-vis_erp.plot(gfp=True, spatial_colors=True, axes=ax[0])
-vis_erp_slow.plot(gfp=True, spatial_colors=True, axes=ax[1])
-ax[0].set_title("Visual ERPs – All Correct Responses")
-ax[1].set_title("Visual ERPs – Slow Correct Responses")
+fig, ax = plt.subplots(3, figsize=(6, 8), layout="constrained")
+vis_erp.plot(gfp=True, spatial_colors=True, axes=ax[0], show=False)
+vis_erp_slow.plot(gfp=True, spatial_colors=True, axes=ax[1], show=False)
+vis_erp_fast.plot(gfp=True, spatial_colors=True, axes=ax[2], show=False)
+
+# Set titles
+ax[0].set_title("All Correct Responses")
+ax[1].set_title("Slow Correct Responses")
+ax[2].set_title("Fast Correct Responses")
+fig.suptitle("Visual ERPs", fontweight="bold")
+
+# Turn of x axes for the first two plots
+ax[0].xaxis.set_visible(False)
+ax[1].xaxis.set_visible(False)
+
+fig
```

Comment:
I suppose @drammock or @larsoner would know an answer to this

Reply:
But note that the tutorials are meant for interactive use and grouped into IPython cells, so …

Reply:
Well, this wouldn't work in the standard Python interactive interpreter (unless interactive Matplotlib mode has been turned on), it only works in a notebook. I was just wondering if `plt.show()` would be the canonical way to show a figure in both a regular interpreter and notebooks.

Reply:
Whatever our expectation is, maybe worth clarifying that explicitly here? https://mne.tools/stable/auto_tutorials/index.html

Reply:
I would use `plt.show()` in sphinx-gallery generated tutorials.

Reply:
I'm happy with either as long as it works. can you make a suggestion or push a change?

Reply:
Just to be clear, you are also relying on a notebook when you do

```
metadata
```

This will output an HTML table, so we cannot replace it with `print(metadata)` (and certainly we don't want to replace it with `IPython.display.display()`. So it's probably OK to assume we're in a notebook I guess (but we should clarify this on the tutorials and examples main pages)?

Reply:
> I would use plt.show() in sphinx-gallery generated tutorials.

I think this is what we've done elsewhere, or just remove the `fig` line